HISTORY.md

```diff
@@ -2,12 +2,22 @@
 
 http://visjs.org
 
```

Comment:
Maybe we should add something like this here:
`For release notes > v6 please visit [vis-timeline/releases at github](https://github.com/visjs/vis-timeline/releases).`